src/mpid/ch4/netmod/ofi/ofi_pre.h

```diff
@@ -171,7 +171,9 @@ typedef struct {
 
 typedef struct {
     fi_addr_t dest;
-#if defined MPIDI_OFI_ENABLE_SCALABLE_ENDPOINTS || defined MPIDI_OFI_ENABLE_RUNTIME_CHECKS
+#if MPIDI_OFI_ENABLE_RUNTIME_CHECKS
+    unsigned ep_idx:MPIDI_OFI_MAX_ENDPOINTS_BITS_SCALABLE;
+#elif MPIDI_OFI_ENABLE_SCALABLE_ENDPOINTS
```

Comment:
Ok, I added one more commit to the beginning of this branch that fixes the offending lookups. @pkcoff - This should work with your use case now. Can you check out this branch and confirm?